### PR TITLE
fix

### PR DESCRIPTION
**Pull request by saeedvaziry**
- Fix logo
- Updater
- Fix history deletion bug
- Show app version in the settings
- Add php8.4
- Update clients
- color highlight for output
- Use OS home directory as settings storage (`.tweakphp`)

### CODE REVIEW

**Comment by saeedvaziry** [comment]
closes #19

File: src/main/updater.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import { autoUpdater } from 'electron-updater'
+import { UpdateInfo } from 'builder-util-runtime'
+import { ipcMain } from 'electron'
+import { window } from './main'
+
+export const init = async () => {
+  autoUpdater.autoDownload = false
```
Comment (luanfreitasdev) [inline src/main/updater.ts:7]: 👋. Unfortunately, auto-updating on macOS is impossible due to the application signature. Still, I did something that "simulates" an auto-update in LaraDumps if you want something like that.

https://www.electron.build/auto-update.html
image

https://github.com/laradumps/app/blob/3.x/src/main/main.ts#L90

https://github.com/laradumps/app/blob/3.x/src/renderer/components/TheAppUpdateInfo.vue#L106

https://github.com/laradumps/app/blob/3.x/src/main/main.ts#L482C13-L482C33

Comment (saeedvaziry) [reply src/main/updater.ts:7]: Thanks for the comment. I think manual update would also work for me. 

Gonna look at Laradumps' code, I think its the best way to proceed 🙏

CleanShot 2024-12-24 at 09 29 18@2x

**Comment by saeedvaziry** [comment]
@luanfreitasdev amazing workaround! worked for a few test I made. I think updating like that is vay simpler than auto update haha.

Thanks

**Pull request merged**